pyproject.toml

```diff
@@ -38,11 +38,10 @@ keywords = [
   "irc",
 ]
 
-# Python spec should be enabled along with bumping `sopel` to 8+
```

Comment:
You removed this comment after implementing only half of it. Next release should drop Sopel 7.1.

Reply:
I read it as "When bumping sopel to 8+, require py>=3.8"